front/inbox.go

```diff
@@ -57,78 +57,80 @@ func dailyPosts(w text.Writer, r *request, day time.Time) {
 	since := now.Add(-time.Hour * 24 * 2)
 
 	rows, err := r.Query(`
-		select notes.object, persons.actor, (case when follows.actor->>'type' = 'Group' then follows.actor else null end) from
-		notes
-		left join (
-			select follows.followed, persons.actor, stats.avg, stats.last from
-			(
-				select followed from follows where follower = $1 and accepted = 1
+		select anotes.object, anotes.actor, anotes.g from
+		(
+			select notes.id, notes.object, notes.author, notes.inserted, notes.cc0, notes.to0, notes.cc1, notes.to1, notes.cc2, notes.to2, follows.followed, follows.actor, (case when follows.actor->>'type' = 'Group' then follows.actor else null end) as g from
+			(select * from notes where inserted >= $4 and inserted < $4 + 60*60*24) notes
```

Comment:
[ ] Check why filtering by insertion time happens so late in the query instead of filtering 99% of rows first, then the complex stuff